packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -52,7 +52,11 @@ const BuySellRow = ({ row: advert }) => {
             <div className='buy-sell-row'>
                 <div
                     className='buy-sell-row__advertiser'
-                    onClick={() => (general_store.is_barred ? null : buy_sell_store.showAdvertiserPage(advert))}
+                    onClick={() =>
+                        general_store.is_barred || !general_store.is_advertiser
```

Comment:
Since in case of the condition evaluating to true, we are returning undefined, can we negate the conditions and just execute the `buy_sell_store.showAdvertiserPage(advert)`?